### PR TITLE
kubevirt-must-gather: use the new command syntax

### PR DESCRIPTION
**Pull request by tiraboschi**
use the new command syntax testing kubevirt-must-gather

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

### CODE REVIEW

File: ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml [hunk 1]
```diff
@@ -68,7 +68,7 @@ tests:
     - as: test-targeted
       cli: latest
       commands: |
-        oc adm must-gather --image="${KMG_IMAGE}" --dest-dir=${ARTIFACT_DIR}/must-gather-output -- gather_vms_details
+        oc adm must-gather --image="${KMG_IMAGE}" --dest-dir=${ARTIFACT_DIR}/must-gather-output --vms_details
```
Comment (machadovilaca) [inline ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml:71]: I think this should be `-- gather --vms_details`, as is `--vms_details` is a flag for `oc adm must-gather` and not for the image  script

File: ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml [hunk 2]
```diff
@@ -110,7 +110,7 @@ tests:
...
```
Comment (machadovilaca) [inline ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml:113]: same as above

**Review by machadovilaca — CHANGES_REQUESTED** [review]

File: ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml (hunk 1 above)
Comment (tiraboschi) [reply ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml]: thanks, amended

**Comment by tiraboschi** [comment]
/pj-rehearse pull-ci-kubevirt-must-gather-main-kubevirt-must-gather-e2e-gcp

**Pull request merged**